YoutubeExplode/Videos/Streams/StreamClient.cs

```diff
@@ -46,6 +61,15 @@ public StreamClient(HttpClient http)
             var url =
                 streamInfoExtractor.TryGetUrl() ??
                 throw new YoutubeExplodeException("Could not extract stream URL.");
+            
+            var signatureParameter = streamInfoExtractor.TryGetSignatureParameter();
+            var signature = streamInfoExtractor.TryGetSignature();
+
+            if (!string.IsNullOrWhiteSpace(signatureParameter) && !string.IsNullOrWhiteSpace(signature))
+            {
+                var spValue = signatureScrambler.Unscramble(signature);
+                url += $"&{signatureParameterp}={spValue}";
+            }
```

Comment:
This is not exactly how it should work, there should be a default parameter in case `signatureParameter` is not set. If I remember correctly, it was `sig` or `s`. Also, Don't update the URL like that, use `Url.SetQueryParameter(...)`.

Reply:
Is guess this what you mean right? 

https://github.com/Tyrrrz/YoutubeExplode/commit/4bdbbf23939db45dd19fba396389527fa3cde857#diff-87f8ee2ac1480a240600a576eaa5a9c96321a809d9d861d5321975c2d44ca75aL37-L51

Reply:
Yes, exactly 🙂